git/libgit2/go.mod

```diff
@@ -0,0 +1,121 @@
+module github.com/fluxcd/pkg/git/libgit2
+
+go 1.17
+
+replace (
+	github.com/fluxcd/pkg/git => ../../git
+	github.com/fluxcd/pkg/gittestserver => ../../gittestserver
+	github.com/fluxcd/pkg/gitutil => ../../gitutil
+	github.com/fluxcd/pkg/http/transport => ../../http/transport
+	github.com/fluxcd/pkg/runtime => ../../runtime
+	github.com/fluxcd/pkg/ssh => ../../ssh
+	github.com/fluxcd/pkg/version => ../../version
+)
+
+// Fix CVE-2022-1996 (for v2, Go Modules incompatible)
+replace github.com/emicklei/go-restful => github.com/emicklei/go-restful v2.16.0+incompatible
+
+// A temporary fork of git2go was created to enable use
+// of libgit2 without thread support to fix:
+// fluxcd/image-automation-controller/#339.
+//
+// This can be removed once libgit2/git2go#918 is merged.
+//
+// The fork automatically releases new patches based on upstream:
+// https://github.com/pjbgf/git2go/commit/d72e39cdc20f7fe014ba73072b01ba7b569e9253
+replace github.com/libgit2/git2go/v33 => github.com/pjbgf/git2go/v33 v33.0.9-nothread-check
+
+require (
+	github.com/Masterminds/semver/v3 v3.1.1
+	github.com/ProtonMail/go-crypto v0.0.0-20220812175011-7fcef0dbe794
+	github.com/fluxcd/gitkit v0.6.0
+	github.com/fluxcd/pkg/git v0.5.0
+	github.com/fluxcd/pkg/gittestserver v0.6.0
+	github.com/fluxcd/pkg/gitutil v0.1.0
+	github.com/fluxcd/pkg/http/transport v0.0.1
+	github.com/fluxcd/pkg/runtime v0.16.2
+	github.com/fluxcd/pkg/ssh v0.5.0
+	github.com/fluxcd/pkg/version v0.1.0
```

Comment:
Just proposing an idea, I need some consensus on this. How about setting the version of all these dependencies that are relative to the package to `v0.0.0`? That way, while reviewing the dependency changes, these can easily be ignored without carefully looking at the import path.
I don't remember where I've seen/read this before, but for some reason I tend to use v0.0.0 by default.

Reply:
You can't import packages that depend on some non-exiting ones, we need to bump the version every time for our consumers.